src/components/input/input.spec.js

```diff
@@ -1,47 +1,65 @@
 describe('md-input-container directive', function() {
+  var $compile, pageScope;
 
-  beforeEach(module('material.components.input'));
-  beforeEach(module('ngAria'));
+  beforeEach(module('ngAria', 'material.components.input'));
+
+  beforeEach(inject(function($injector) {
+    $compile = $injector.get('$compile');
+
+    var $rootScope = $injector.get('$rootScope');
+    pageScope = $rootScope.$new();
```

Comment:
Should these be merged into a single line?
`pageScope = $injector.get('$rootScope').$new()`